nautobot/extras/tables.py

```diff
@@ -320,8 +332,8 @@ class ApprovalWorkflowStageTable(BaseTable):
         orderable=False,
         verbose_name="Actions Needed",
     )
-    state = ChoiceFieldColumn()
-    actions = ApprovalButtonsColumn(ApprovalWorkflowStage, buttons=("approve", "deny", "comment"))
```

Comment:
Did we remove the `comment` button entirely, or are there still places it's used? If the former, are there views and/or templates that can also be removed?

Reply:
I removed the `comment` button entirely from the original `ApprovalButtonsColumn`